.github/workflows/release.yml

```diff
@@ -88,7 +88,7 @@ jobs:
         - os: macos-11
           dist-args: --artifacts=local --target=aarch64-apple-darwin --target=x86_64-apple-darwin
           install-dist: curl --proto '=https' --tlsv1.2 -LsSf https://github.com/axodotdev/cargo-dist/releases/download/v0.0.7/cargo-dist-installer.sh | sh
-        - os: ubuntu-22.04
+        - os: ubuntu-20.04
           dist-args: --artifacts=local --target=x86_64-unknown-linux-gnu
```

Comment:
Why go back to 20.04?

Reply:
I **think** that with the openssl dependency vendored we should be able to run on 20 and 22